### PR TITLE
Add supported range warnings

### PR DESCRIPTION
**Pull request by jamesdaniels**
* Adding supported range warnings to NextJS, Angular, Nuxt, Astro, and Vite
* Use this supported range for the bootstrap scripts
* Remove supported version from the NextJS doc, that was unnecessarily scaring users away from Firebase IMO

Example NextJS:
Screenshot 2023-12-02 at 8 55 57 PM

Example Svelte:
Screenshot 2023-12-02 at 8 53 49 PM

### CODE REVIEW

File: src/frameworks/sveltekit/index.ts [hunk 1]
```diff
@@ -11,7 +11,8 @@ export const name = "SvelteKit";
 export const support = SupportLevel.Experimental;
 export const type = FrameworkType.MetaFramework;
 export const discover = viteDiscoverWithNpmDependency("@sveltejs/kit");
-export { getDevModeHandle } from "../vite";
+
+export { getDevModeHandle, supportedRange } from "../vite";
 
 export async function build(root: string) {
```
Comment (jamesdaniels) [inline src/frameworks/sveltekit/index.ts:17]: sveltekit should probably have it's own supported range

File: src/frameworks/utils.ts [hunk 2]
```diff
@@ -360,21 +361,36 @@ export function relativeRequire(dir: string, mod: string) {
...
+  const allButLast = opts.slice(0, -1);
   return `${allButLast.join(`${separator} `)}${separator} ${conjunction} ${lastElement}`;
 }
 
-export function frameworksCallToAction(message: string, docsUrl = DEFAULT_DOCS_URL, prefix = "") {
-  return `${prefix}${message}
+export function frameworksCallToAction(
+  message: string,
+  docsUrl = DEFAULT_DOCS_URL,
+  prefix = "",
```
Comment (leoortizz) [inline src/frameworks/utils.ts:377]: nit: maybe prefix could be handled internally in the function based on the params? e.g.:
```ts
const prefix = framework ? "   " : "";
```
just thinking about reducing the parameters here

File: src/frameworks/sveltekit/index.ts (hunk 1 above)
Comment (leoortizz) [reply src/frameworks/sveltekit/index.ts:17]: indeed but maybe after 2.0 is released? It seems like it won't be released anytime soon: https://github.com/sveltejs/kit/milestone/5

**Review by leoortizz — APPROVED** [review]
LGTM. Left only one suggestion.

File: src/frameworks/sveltekit/index.ts (hunk 1 above)
Comment (jamesdaniels) [reply src/frameworks/sveltekit/index.ts:17]: Fair enough

File: src/frameworks/utils.ts (hunk 2 above)
Comment (jamesdaniels) [reply src/frameworks/utils.ts:377]: i'll refactor this in a follow on

**Pull request merged**